lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -47,6 +47,11 @@ states:
       next:
         targetState: POST_APP_DOC_CHECK_INTERNATIONAL_SUCCESS_PAGE
 
+  IDENTITY_DOCUMENT_START:
+    events:
+      next:
+        targetState: IDENTITY_START_PAGE
```

Comment:
I think we're best going through `RESET_SESSION_IDENTITY` first? we don't really want the existing VCs hanging about